src/main/java/frc/robot/commands/TurnRelativeCommand.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.DriveSubsystem;
+
+public class TurnRelativeCommand extends Command {
+	private DriveSubsystem m_driveSubsystem;
+	private double m_angle;
+	private PIDController m_controller;
+
+	/**
+	 * Creates a new TurnRelativeCommand.
+	 * 
+	 * @param subsystem The drive subsystem.
+	 * @param angle     The angle in degrees.
+	 */
+	public TurnRelativeCommand(DriveSubsystem subsystem, double angle) {
+		m_driveSubsystem = subsystem;
+		m_angle = angle;
+		m_controller = new PIDController(0.0075, 0, 0);
+		m_controller.enableContinuousInput(0, 360);
+		m_controller.setTolerance(5);
```

Comment:
May want to lower this and keep tuning, absolute turn may be the better use of our time for right now though